lib/pliny/librato/metrics/backend.rb

```diff
@@ -56,7 +56,10 @@ def start_timer
           @timer = Thread.new do
             loop do
               sleep interval
-              flush_librato
+              begin
+                flush_librato
+              rescue # allow for periodic librato errors to recover
+              end
```

Comment:
Should we at least log this with some sort of countable message so we can monitor better?